extras/dummy_generator/generator.py

```diff
@@ -415,3 +437,43 @@
 
         # Bulk-create the weight entries
         WeightEntry.objects.bulk_create(new_entries)
+
+# Nutrition Generator
+if hasattr(args, 'number_nutrition_plans'):
+    print("** Generating {0} nutrition plan(s) per user".format(args.number_nutrition_plans))
+
+    if args.add_to_user:
+        userlist = [User.objects.get(pk=args.add_to_user)]
+    else:
+        userlist = [i for i in User.objects.all()]
+
+    # Load all ingredients to a list
+    ingredientList = [i for i in Ingredient.objects.all()]
```

Comment:
While this is a script that won't get used often, I'm not sure about loading all the ingredients at once. I'd perhaps do something like this (even if this is also not a perfect solution):

```python
ingredientList = [i for i in Ingredient.objects.order_by('?').all()[:100]]
```